docs/learn-faq.md

```diff
@@ -6,80 +6,327 @@ sidebar_label: Frequently Asked Questions (FAQs)
+token holders. The Sudo account will now initiate an upgrade that erases the
+Sudo logic from the chain entirely and replaces it with the stakeholder
+governance modules. From henceforward, the network is in the hands of the token
+holders and no longer under control of an authority.
+
+The final step to transition to full-functioning Polkadot is the enabling of
+transfer functionality. The community will need to propose a new upgrade and vote
+upon it in order to accomplish this. If the vote to enable transfers passes, then
+it will be enacted shortly thereafter. When this takes place, Polkadot will
+be fully live and move out of it's Beta release label.
+
+### How many validators will be validating at Polkadot Beta launch?
+
+Initially only a few authorized validators will be securing the network. However,
+as [detailed in the answer above](#what-is-the-launch-process-of-polkadot-beta),
+the network will right away be available to vaidators that want to register
+their intention to validate. The transition to Proof-of-Stake will largely
+depend on when a sufficient number of validators have registered and are ready
+to take over the security of the network. This number can be as low as 50 but
+probably closer to 100.
+
+## Validators
 
 ### How do I apply to be a validator?
 
-See ("How To Validate on Kusama")[maintain-guides-how-to-validate-kusama.md].  Note that just because you run a validator does not mean you will necessarily be producing blocks; only a certain number of validators will be part of the active validator set, based on the amount of stake they can attract and the results of the (Phragmen election)[learn-phragmen].
+There is no central authority that decides on validators, so there is not per se
+an _application_ that you can fill out. Registering as a validator is
+permissionless, in order to become one you must only set up a validator node and
+mark your intention to validate on chain. For detailed instruction on how to do
+this you can consult the [wiki guide](maintain-guides-how-to-validate-kusama.md)
+on validating for Kusama.
+
+However, once you've set up a validator and have registered your intention it
+does not mean that you will be included in the _active set_ right
+away. The validators are elected to the active set based on the results of an
+election algorithm known as [Phragmen's method](learn-phragmen). Phragmen's
+tries to accomplish two things, 1) select `n` members from a larger set based
+on stake-weighted votes and 2) equalize the stake backing each validator as 
+much as possible.
+
+You will likely want to campaign your validator to the community in order to
+get more backing. You are looking for _nominators_ that will put up their tokens
+to increase the stake for your validator. Besides this, for validators who
+cannot acquire the minimum stake from the community, Parity and Web3 Foundation
+run a joint program called [Thousand Validators][thousand validators] that will
```

Comment:
Is this the right link? `[thousand validators]`

It does not look properly formatted.

Reply:
It's a [reference-style link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links). Usually I find these easier for longer documents since we can collect all the external links at the bottom of the document.